README.md

```diff
@@ -74,13 +122,23 @@ Visit [http://localhost:3000/](http://localhost:3000/) to see the locally runnin
 ## Testing
 Make sure you have the same version of Node.js as specified in `package.json`, otherwise you may have trouble running the front end tests.
 
-Run tests with:
+If you are using Docker for development, run:
+```shell
+docker-compose run app npm test
+```
+
+If you are running Checklistomania locally, run:
 ```shell
 npm test
 ```
 
 Run eslint either via a code editor plugin (such as Atom's [linter-eslint](https://github.com/AtomLinter/linter-eslint)),
-or from the command line with:
+or from the command line. If you are using Docker, run:
+```shell
+docker-compose run app npm run eslint
```

Comment:
That's not a bad idea. I kinda like keeping everything that needs to run in npm scripts though.